packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -11,14 +11,10 @@ import { Computation } from '../../../types/visualization';
 import SingleSelect from '@veupathdb/coreui/lib/components/inputs/SingleSelect';
 import { useMemo } from 'react';
 import { ComputationStepContainer } from '../ComputationStepContainer';
+import './Plugins.scss';
+import { makeClassNameHelper } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
 
-export const sharedConfigCssStyles = {
-  display: 'flex',
-  gap: '0 2em',
-  padding: '1em 0',
-  alignItems: 'center',
-  marginLeft: '3em',
-};
+const cx = makeClassNameHelper('StepOneConfigurationContainer');
```

Comment:
Would this class name be confusing if we have a step 1 elsewhere that doesn't have to do with configuring a compute? Do you think this is general enough to be just `StepConfigurationContainer`? Or maybe specific to computes like `AppStepConfigurationContainer`?

Reply:
Hmm, these are good questions. I didn't suspect it would apply beyond app/plugin configurations, which is why I named the file `Plugins.scss`. While not convinced of my naming (ever, really), I think the fact that these styles live in `Plugins.scss` may be enough to avoid confusion? That said, I think `AppStepConfigurationContainer` may be a better name anyway.